tests/test_optimizer.py

```diff
@@ -0,0 +1,106 @@
+# ------------------------------------------------------------------------------
+# Copied from https://github.com/open-mmlab/mmediting
```

Comment:
although it is reasonable and okay, not sure if it is a good idea to start this notice within the mm-series.

Reply:
cc. @hellock , we can borrow code freely in mm-*, right?

Reply:
OK, I will drop this notice.

Reply:
Yes we can use codes from other codebases in OpenMMLab without this notice.